crnk-core/src/main/java/io/crnk/core/queryspec/DefaultQuerySpecDeserializer.java

```diff
@@ -172,24 +185,19 @@ public QuerySpec deserialize(ResourceInformation resourceInformation, Map<String
 					deserializeFields(querySpec, parameter);
 					break;
 				case page:
-					deserializePage(querySpec, parameter);
+					pageParameters.put(parameter.pageParameter, parameter.getValues());
 					break;
 				default:
 					deserializeUnknown(querySpec, parameter);
 			}
-
+		}
+		if (resourceInformation.getPagingBehavior() != null) {
```

Comment:
I would say BadRequestException is unnecessary here since:

1. Non-existing PagingBehavior isn't really a bad request issue

2. We should always have default impl which is added at CrnkBoot

Reply:
I'm ok with the pagingBehavior being null if no paging is supported. But if there is no paging ANDa paging parameter was provided, then it should throw an exception?

Reply:
Well, I added the check, but tbh still not sure about. The reason for the initial check is mostly for custom modules such as crnk-client, the default server-side behavior in general should always have default instance of PagingPehavior